storybook/qmlTests/tests/tst_TokenSelector.qml

```diff
@@ -0,0 +1,221 @@
+import QtQuick 2.15
+import QtTest 1.15
+import QtQml 2.15
+
+import Models 1.0
+
+import AppLayouts.Wallet.controls 1.0
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.adaptors 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    QtObject {
+        id: d
+
+        readonly property var flatNetworks: NetworksModel.flatNetworks
+        readonly property var assetsStore: WalletAssetsStore {
+            id: thisWalletAssetStore
+            walletTokensStore: TokensStore {
+                plainTokensBySymbolModel: TokensBySymbolModel {}
+            }
+            readonly property var baseGroupedAccountAssetModel: GroupedAccountsAssetsModel {}
+            assetsWithFilteredBalances: thisWalletAssetStore.groupedAccountsAssetsModel
+        }
+
+        readonly property var adaptor: TokenSelectorViewAdaptor {
+            assetsModel: d.assetsStore.groupedAccountAssetsModel
+            flatNetworksModel: d.flatNetworks
+            currentCurrency: "USD"
+
+            Binding on searchString {
+                value: controlUnderTest ? controlUnderTest.searchString : ""
+                restoreMode: Binding.RestoreNone
+            }
+        }
```

Comment:
As it's a unit test, the component should be tested in isolation. `TokenSelectorViewAdaptor` should be tested in it's own unit test. Similar rule should be applied to storybook page - dedicated page for `TokenSelector` and `TokenSelectorViewAdaptor`, with auxiliary controls modifying only properties of given component.